web/src/compose.js

```diff
@@ -169,7 +178,137 @@ export function send_message_success(request, data) {
     }
 }
 
-export function send_message(request = create_message_object()) {
+let split_long_messages_banner_shown = false;
+
+export function update_split_long_messages_info_banner() {
+    if (check_need_to_split_long_messages()) {
+        show_split_long_messages_info_banner();
+        compose_ui.autosize_textarea($("textarea#compose-textarea"));
+    } else {
+        clear_split_long_messages_info_banner();
+    }
+}
+
+const SPLIT_DELIMITER = "\n\n\n";
+/**
+ * checks whether the message actually needs to be split, as it is possible that `split_long_messages` is enabled but the message doesn't contain the delimiter.
+ */
+function check_need_to_split_long_messages() {
+    if (!is_split_long_messages_enabled()) {
+        return false;
+    }
+    const content = compose_state.message_content();
+    return content.includes(SPLIT_DELIMITER);
+}
```

Comment:
Hmm, `check_need_to_split_long_messages` feels like the wrong name, because we don't check whether the message is long, just whether it contains the split syntax. 

I think `compose.currently_composing_multiple_messages()` would be a reasonable function name for this boolean purpose. Though given how it seems to be used from callers that have always fetched `compose_state.message_content`, maybe `compose.will_split_into_multiple_messages(content)` would be even clearer.